tests/failpoints/cases/test_storage.rs

```diff
@@ -509,7 +509,15 @@ fn test_pipelined_pessimistic_lock() {
 
 #[test]
 fn test_async_commit_prewrite_with_stale_max_ts() {
-    let mut cluster = new_server_cluster(0, 2);
+    test_kv_format_impl!(test_async_commit_prewrite_with_stale_max_ts_impl);
```

Comment:
Suggest to write as other test cases do (see [here](https://github.com/tikv/tikv/blob/a68a44e09dd4ef7de59db22b11593e7abd4e94df/src/storage/mod.rs#L7039)):
```
test_async_commit_prewrite_with_stale_max_ts_impl::<ApiV1>();
test_async_commit_prewrite_with_stale_max_ts_impl::<ApiV2>();
```

Reply:
fixed, thanks for your suggestion, PTAL~